images/runtime/python/generateDockerfiles.sh

```diff
@@ -21,14 +21,17 @@ declare -r ORYX_PYTHON_IMAGE_BASE_PLACEHOLDER="%BASE_TAG%"
 
 source "$PYTHON_VERSIONS_PATH"
 
-declare -r PYTHON_BUSTER_VERSION_ARRAY=($PYTHON310_VERSION $PYTHON39_VERSION)
+declare -r PYTHON_BULLSEYE_VERSION_ARRAY=($PYTHON310_VERSION)
+declare -r PYTHON_BUSTER_VERSION_ARRAY=($PYTHON39_VERSION)
```

Comment:
For 3.9.12, we'll need the Bullseye image as well, right? Do you prefer to track that separately?

Reply:
That's one is a separate task from the purpose of this PR. I can add that one in a separate PR.